installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -0,0 +1,116 @@
+[id="installing-gcp-user-infra-vpc"]
+= Installing a cluster on GCP using Deployment Manager templates
```

Comment:
Shared VPC is missing in the title. Suggest to update it to Installing a cluster in to a shared VPC on GCP using Deployment Manager templates.